### ISSUE TITLE
Uptime is calculated incorrectly

### ISSUE THREAD

**Issue by medvednikovtwolove** [vlang/gitly#151]
Uptime shown in admin interface is completely wrong, for example:
![image](https://user-images.githubusercontent.com/104035389/164894408-a27ac267-92d5-4cbc-8520-675019d1fcb9.png)
(you can also see here that auth issue mentioned in #150 is not fixed)

I believe it has something to do with the following code:

```v
pub fn (mut app App) running_since() string {
	dur := time.now().unix - app.started_at
	seconds := dur % 60
	minutes := int(math.floor(dur / 60)) % 60
	hours := int(math.floor(minutes / 60)) % 24
	days := int(math.floor(hours / 24))
	return '$days days $hours hours $minutes minutes and $seconds seconds'
}
```

Taking into account that `app.started_at` was always zero whenever I checked, it should show a really enormous uptime, but in fact it always shows that Gitly is up for less than a hour.

**Comment by medvednikov** [comment]
No it's correct, it's been restarted.

Any hints on the auth issue?

**Comment by medvednikovtwolove** [comment]
> No it's correct, it's been restarted.

No, it's not. It just shows number of minutes and seconds from the beginning of Unix epoch. I took this screenshot just a couple of seconds ago:

![image](https://user-images.githubusercontent.com/104035389/164895683-ad76a8cd-367e-49aa-b2d8-d357c54ee279.png)

It ticks from 0 again each hour.

**Comment by walkingdevel** [comment]
Fixed #156
